local/languages/finna/en-gb.ini

```diff
@@ -258,7 +258,8 @@ due_date_reminder_one_day = "One day before due date"
 due_date_reminder_three_days = "Three days before due date"
 due_date_reminder_two_days = "Two days before due date"
 due_date_email_error = "Failed to check due dates of your loans"
-due_date_email_login_error = "We couldn't verify all of your library cards for an account you last used %%lastLogin%% by logging in with %%loginMethod%%. Due date reminders for these cards have been disabled. If you don't use this account, you can ignore this message. Otherwise you can turn the reminders back on after logging in to %%serviceName%%."
+due_date_email_login_error = "We were not able to find data on all your library cards. The due date reminders of these cards have now been deactivated. You can reactivate your due date reminders after signing in to the %%serviceName%% service."
+due_date_email_login_last = "If you no longer use this account, this message does not require any further actions. You last used this account on %%lastLogin%% by logging in with your %%loginMethod%%."
```

Comment:
"your" ei toimi %%loginMethod%%:n kanssa. ks. due_date_login_method_*. Jälkimmäisiä käännöksiä ei voi muuttaa ilman ongelmia taaksepäin yhteensopivuuden kanssa.